admin/src/components/connect-banner.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from '@atlaskit/image';
+import { css } from '@emotion/react';
+
+export function ConnectBanner() {
+	return (
+		<div
```

Comment:
nit: it's nothing major but Atlaskit also now has support for [primitives](https://atlassian.design/components/primitives/overview/)! Might help reduce some of the need for custom styles

Reply:
yeah I briefly looked into using Box/Stack/Inline but was thrown off by parts of the API. Maybe worth moving to in the future? Heh, I originally started with tailwind but didn't want to spend time debugging it clashing with the CSS reset we get from Atlassian.

Reply:
Makes sense, it is still in beta so no stress with this! Tailwind is also my go to <3